openedx/features/philu_utils/backend_storage.py

```diff
@@ -39,8 +39,16 @@ def _s3_client(self):
             aws_secret_access_key=self.aws_secret_access_key,
         )
 
-    def _get_available_name(self, name, max_length=None):  # pylint: disable=unused-argument
-        """Truncate name if it is more than allowed max length and get alternate name if name already exists"""
+    def _get_available_name(self, name):
+        """
+        Get alternate name if name already exists
+
+        Args:
```

Comment:
For future reference, let's stick to edx recommendation and write "Arguments" instead of "Args"

Reply:
Fixed in this PR as well